app/src/main/java/de/danoeh/antennapod/ui/swipeactions/SwipeActionsDialog.java

```diff
@@ -147,29 +151,52 @@ private void showPicker(SwipeactionsRowBinding view, int direction) {
         MaterialAlertDialogBuilder builder = new MaterialAlertDialogBuilder(context);
         builder.setTitle(direction == LEFT ? R.string.swipe_left : R.string.swipe_right);
 
-        SwipeactionsPickerBinding picker = SwipeactionsPickerBinding.inflate(LayoutInflater.from(context));
-        builder.setView(picker.getRoot());
+        LayoutInflater inflater = LayoutInflater.from(context);
+        View pickerView = inflater.inflate(R.layout.swipeactions_picker, null);
```

Comment:
Please keep using ViewBinding instead of switching back to manual layout inflation with `findViewById`